main/server/validations/userValidations.js

```diff
@@ -0,0 +1,22 @@
+const t = require('io-ts')
+const common = require('./common')
+const { patterns } = require('../../../core/helpers')
+
+/**
+ * @typedef {{
+ *   name: string,
+ *   society?: string,
+ *   job?: string
+ * }} ProfilPatch
+ */
+
+/** @type {Decoder<ProfilPatch>} */
+const userPatchType = t.exact(t.type({
```

Comment:
We ensure to have an exact model and avoid invalid data / data injection trough the rest of the process.